src/components/CustomChannelComponent.tsx

```diff
@@ -187,6 +198,8 @@ export function CustomChannelComponent() {
     return getBotWelcomeMessages(allMessages, botId);
   }, [messageCount]);
 
+  if (!channel || !botUser) return null;
```

Comment:
Are there any side effects on the UI?

Reply:
ui will be empty. but this is safer because without it, run time error happens

Reply:
Isn't it okay without this in develop branch? Please check this one before you merge 👍

Reply:
I added for safety but not tested. ill put back